### PR TITLE
[CUBRIDMAN-136] Reinforce result items of SHOW INDEX Capacity

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CUBRIDMAN-136

* The description of the result item of the added "Show index capacity" syntax is reinforced.
   Information about the overflow page.
   
* Code changes related to this fix can be found at the link below.
   [CBRD-24425]

### CODE REVIEW

**Review by sungc-cubrid — APPROVED** [review]

**Review by hyunikn — APPROVED** [review]

File: en/sql/query/show.rst [hunk 1]
```diff
@@ -1431,15 +1431,21 @@ Total_value                         INT             Total number of values store
 Avg_num_value_per_key               INT             Average number of values (OIDs) per key
 Num_leaf_page                       INT             Leaf page count
 Num_non_leaf_page                   INT             NonLeaf page count
+Num_ovf_page                        INT             OID overflow page count
```
Comment (mhoh3963) [inline en/sql/query/show.rst]: The description seems to be difficult for users to understand.
So, how about changing OID overflow to leaf's overflow?

**Review by beyondykk9 — APPROVED** [review]

File: en/sql/query/show.rst (hunk 1 above)
Comment (ctshim) [reply en/sql/query/show.rst]: Received comments and edited.
Thank you.

File: ko/sql/query/show.rst [hunk 2]
```diff
@@ -1438,15 +1438,21 @@ Total_value                         INT             트리에 저장된 값의 
...
 Avg_key_len                         INT             평균 키 길이
 Avg_rec_len                         INT             평균 페이지 레코드 길이
 Total_space                         VARCHAR(64)     인덱스에 의해 점유되는 전체 공간
-Total_used_space                    VARCHAR(64)     인덱스의 전체 사용 공간
-Total_free_space                    VARCHAR(64)     인덱스의 전체 여유 공간
-Avg_num_page_key                    INT             단말 노드 페이지에서 페이지 당 평균 키 개수
-Avg_page_free_space                 VARCHAR(64)     페이지 당 평균 여유 공간
+Total_used_space_non_ovf            VARCHAR(64)     할당된 페이지에서 사용된 총 공간(단말 노드의 오버플로우 페이지는 제외)
+Total_free_space_non_ovf            VARCHAR(64)     할당된 페이지에서 사용되지 않은 총 공간(단말 노드의 오버플로우 페이지는 제외)
+Total_used_space_ovf                VARCHAR(64)     단말 노드의 오버플로우에 할당된 페이지에서 사용된 총 공간
```
Comment (mhoh3963) [inline ko/sql/query/show.rst]:
```suggestion
Total_used_space_ovf                VARCHAR(64)     단말 노드의 할당된 오버플로우 페이지에서 사용된 총 공간
```

File: ko/sql/query/show.rst [hunk 3]
```diff
@@ -1438,15 +1438,21 @@ Total_value                         INT             트리에 저장된 값의 
...
+Total_free_space_ovf                VARCHAR(64)     단말 노드의 오버플로우에 할당된 페이지에서 사용되지 않은 총 공간
```
Comment (mhoh3963) [inline ko/sql/query/show.rst]:
```suggestion
Total_free_space_ovf                VARCHAR(64)     단말 노드의 할당된 오버플로우 페이지에서 사용되지 않은 총 공간
```

File: en/sql/query/show.rst [hunk 4]
```diff
@@ -1431,15 +1431,21 @@ Total_value                         INT             Total number of values store
...
 Total_space                         VARCHAR(64)     Total space occupied by index
-Total_used_space                    VARCHAR(64)     Total used space in index
-Total_free_space                    VARCHAR(64)     Total free space in index
-Avg_num_page_key                    INT             Average page key count (in leaf pages)
-Avg_page_free_space                 VARCHAR(64)     Average page free space
+Total_used_space_non_ovf            VARCHAR(64)     Total space used in allocated pages (excluded leaf's overflow pages)
+Total_free_space_non_ovf            VARCHAR(64)     Total space unused in allocated pages (excluded leaf's overflow pages)
+Total_used_space_ovf                VARCHAR(64)     Total space used in pages allocated to leaf's overflow
+Total_free_space_ovf                VARCHAR(64)     Total space unused in pages allocated to leaf's overflow
+Avg_num_key_per_page_non_ovf        INT             Average page key count (in leaf pages)
```
Comment (mhoh3963) [inline en/sql/query/show.rst]:
```suggestion
Avg_num_key_per_page_non_ovf        INT             Average page key count in leaf pages
```

File: en/sql/query/show.rst [hunk 5]
```diff
@@ -1431,15 +1431,21 @@ Total_value                         INT             Total number of values store
...
+Avg_free_space_per_page_non_ovf     VARCHAR(64)     Average page free space
+Avg_num_ovf_page_per_key            INT             Average page key count (in leaf pages)
```
Comment (mhoh3963) [inline en/sql/query/show.rst]:
```suggestion
Avg_num_ovf_page_per_key            INT             Average page key count in leaf pages
```

File: en/sql/query/show.rst [hunk 6]
```diff
@@ -1431,15 +1431,21 @@ Total_value                         INT             Total number of values store
...
+Avg_free_space_per_page_ovf         VARCHAR(64)     Average page free space
```
Comment (mhoh3963) [inline en/sql/query/show.rst]:
```suggestion
Avg_free_space_per_page_ovf         VARCHAR(64)     Average page free space in leaf's overflow pages
```

Comment (ctshim) [reply en/sql/query/show.rst]: Thank you.

**Review by hgryoo — APPROVED** [review]

**Review by hiclass — APPROVED** [review]

**Review by mhoh3963 — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Pull request merged**